packages/web/src/app/doc-pages/brand/icon-doc/icon-preview/icon-preview-details/icon-preview-details.component.html

```diff
@@ -0,0 +1,53 @@
+<i [ngClass]="['e-icon', 'e-icon--' + selectedIcon()?.title]" aria-hidden="true"></i>
+
+<div class="info">
+  <div class="text">
+    <h3 class="e-text-md-strong e-m-0">{{ selectedIcon()?.pretty }}</h3>
+    <span class="e-text-sm e-color-text-2 e-none-sm"
+      >e-icon e-icon--{{ selectedIcon()?.title }} e-icon--sm</span
+    >
+  </div>
+  <div class="actions">
+    <app-copy
+      [style.padding-bottom]="'3px'"
```

Comment:
This could also be defined as `style="padding-bottom: 3px;"`. A CSS-class could also be appropriate